command/audit_list.go

```diff
@@ -94,13 +94,13 @@ func (c *AuditListCommand) Run(args []string) int {
 		return 2
 	}
 
-	if len(audits) == 0 {
-		c.UI.Output(fmt.Sprintf("No audit devices are enabled."))
-		return 0
-	}
-
 	switch Format(c.UI) {
 	case "table":
+		if len(audits) == 0 {
+			c.UI.Output(fmt.Sprintf("No audit devices are enabled."))
+			return 0
```

Comment:
`audit list` returns `0` as error code but literally any other list command returns `2`. Should we align that?

Reply:
Huh. What does normal vault list do for empty?

Reply:
https://github.com/hashicorp/vault/blob/master/command/list.go#L87
https://github.com/hashicorp/vault/blob/master/command/namespace_list.go#L76

All other list commands return the error code `2` only the audit list command returns `0`. I'm thinking about aligning the return codes and also return the error code `2` here?

Reply:
Sure, return 2